src/components/GraphiQL.js

```diff
@@ -16,10 +16,10 @@ import { ToolbarButton } from './ToolbarButton';
 import { ToolbarGroup } from './ToolbarGroup';
 import { ToolbarMenu, ToolbarMenuItem } from './ToolbarMenu';
 import { ToolbarSelect, ToolbarSelectOption } from './ToolbarSelect';
+import { PluginsPane } from './PluginsPane';
 import { QueryEditor } from './QueryEditor';
 import { VariableEditor } from './VariableEditor';
 import { ResultViewer } from './ResultViewer';
-import { AnalysisViewer } from './AnalysisViewer';
```

Comment:
I've removed `AnalysisViewer` for now

Reply:
Are you envisioning using the component code for this at some point? We should probably just remove it for now as well and re-add if necessary.